recipes-containers/docker/docker.inc

```diff
@@ -77,23 +78,26 @@ do_compile() {
 
 	export DISABLE_WARN_OUTSIDE_CONTAINER=1
 
-	cd ${S}/src/import/
-
 	# this is the unsupported built structure
 	# that doesn't rely on an existing docker
 	# to build this:
 	VERSION="${DOCKER_VERSION}" DOCKER_GITCOMMIT="${DOCKER_COMMIT}" ./hack/make.sh dynbinary
 
-        # build the cli
+	# build the cli
 	cd ${S}/src/import/.gopath/src/github.com/docker/cli
 	export CFLAGS=""
 	export LDFLAGS=""
 	export DOCKER_VERSION=${DOCKER_VERSION}
-	VERSION="${DOCKER_VERSION}" DOCKER_GITCOMMIT="${DOCKER_COMMIT}" make dynbinary
+	VERSION="${DOCKER_VERSION}" DOCKER_GITCOMMIT="${DOCKER_COMMIT}" make binary
 
+	cd ${S}/src/import
```

Comment:
It looks like this line was moved here from earlier in the file. However, given that this cd command is immediately followed by another cd command, shouldn't it just be removed instead? If for some reason you don't want to remove it, should it just stay in its previous location?

Reply:
Move back to it's previous location